docs/tutorials/losses_triplet.ipynb

```diff
@@ -144,19 +144,6 @@
         "## Setup"
```

Comment:
<p>could you remove redundant line 2 and 3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>